jaxrs-api/src/main/java/jakarta/ws/rs/client/ClientRequestContext.java

```diff
@@ -192,6 +193,29 @@ public default boolean hasProperty(String name) {
      */
     public String getHeaderString(String name);
 
+    /**
+     * Checks whether a header with a specific name and value (or item of the comma-separated value list) exists.
+     *
+     * Each single header value is converted to String using a {@link jakarta.ws.rs.ext.RuntimeDelegate.HeaderDelegate} if one
```

Comment:
For consistency with other text, "Each single header value ..." -> "Each non-string header value ...". We should not use a delegate if already a String.

Reply:
This is the original text taken from `#getHeaderString`, so do you want to change *that* text, too? I deliberately used it unchanged to prevent any inconsistencies, actually.

Reply:
Well, I think `#getStringHeaders` in the same class is more precise, isn't it? Seems important to use the _non-string_ qualifier.

Reply:
Sure, but `#getStringHeader(String name)` unfortunately is *missing* the `non-string` token, hence I was asking if you want to thange it *there*, too. Do you?

Reply:
Assuming you *would* add the token `non-string` to `#getStringHeader(String name)`, too, I added it two both code locations in https://github.com/jakartaee/rest/pull/1066/commits/eb18e39ce3824724f33116f01c4a0cddc64bfbb7.

Ok?